package.json

```diff
@@ -87,5 +87,8 @@
     "stylelint-config-terra": "^2.0.0",
     "terra-dev-site": "^3.0.0",
     "terra-toolkit": "^4.16.0"
+  },
+  "dependencies": {
+    "keycode-js": "^1.0.4"
```

Comment:
We will need to remove this dependence from this package.json file and add it as a dependency in the `terra-clinial-data-grid` package.json.

When we release the components in `terra-clinical` repo, the package.json file defined in the component's directory will be shipped and used in the install process. This means, when someone installs `terra-clinial-data-grid`, the dependencies defined in the `terra-clinical-data-gird`'s package.json will be the only dependencies installed with the component. The devDependencies defined in this package.json (the root package.json) are only used for install the repo for development purposes because we don't actually release the monorepo.

Reply:
This will need to be added to `terra-item-collection` as well.

Reply:
https://github.com/cerner/terra-clinical/pull/479/commits/09ba9949452822abb1e3446957588da693d98182#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L90